service-discovery/stork/src/main/resources/application.properties

```diff
@@ -1 +1,14 @@
-# application properties should be here
+quarkus.http.port=18080
```

Comment:
I don't like this. `quarkus.http.port` is usually overridden in tests. Is there any reason to have it hardcoded?